test/extensions/filters/http/ext_proc/tracer_test_filter.cc

```diff
@@ -116,9 +116,9 @@ class Span : public Tracing::Span {
   ExpectedSpansSharedPtr expected_spans_;
 
   std::map<std::string, std::string> tags_;
-  bool context_injected_;
-  bool sampled_;
-  bool finished_;
+  bool context_injected_{false};
+  bool sampled_{false};
+  bool finished_{false};
```

Comment:
I [noticed](https://github.com/envoyproxy/envoy/actions/runs/12467778354/job/34800128408#step:17:4165) we still have some flake tests in CI related to extproc and tracing and wonder if might be it.